src/activity_item_handling.cpp

```diff
@@ -183,9 +183,6 @@ struct act_item {
 };
 
 // TODO: Deliberately unified with multidrop. Unify further.
```

Comment:
I'm fairly sure that comment can go as well. On one hand because it was referring to the usings and on the other hand because it's 9 years old and absolutely unclear, if any further unification is still necessary and if so how.